src/components/FeatureViews/Scheduler/Scheduler.js

```diff
@@ -293,7 +381,8 @@ class Scheduler extends Component {
                 session["title"] = session.title;
                 session["description"] = session.description ? session.description : "";
                 session["type"] = this.props.courses[session.course_id].type;
-                session["resourceId"] = this.props.courses[session.course_id].room_id;
+                session["resourceId"] = this.props.courses[session.course_id].room_id ? this.props.courses[session.course_id].room_id : 1;
```

Comment:
Can be shortened to: `session["resourceId"] = this.props.courses[session.course_id].room_id || 1;`